### PR TITLE
Add autograd_gamma a dependency for lifelines

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@CamDavidsonPilon are you okay with being a maintainer on this recipe as well?

**Comment by CamDavidsonPilon** [comment]
Hey @mariusvniekerk - yes absolutely 👍

File: recipes/autograd_gamma/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3b4349cb415bd6e28dd2fac5055e34de1b6c87fe711757a0e42a84bd650fba35"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (scopatz) [inline recipes/autograd_gamma/meta.yaml]: Can this be `noarch: python`?

**Comment by scopatz** [comment]
Many CI failures :frowning:

**Comment by ericmjl** [comment]
cc: @conda-forge/python I noticed that this package's has been merged, but is not discoverable on conda-forge whether by the `autograd-gamma` or `autograd_gamma`. PR #10005 depends on this package; I'm wondering if there's something I might be able to do to get the build to work?

**Comment by ocefpaf** [comment]
AzurePipelines sometimes fails on the feedstock. Investigating, if that is the case I'll re-start it.

**Comment by ocefpaf** [comment]
@ericdill CIs restarted. Watch https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=85259 for the package.

PS: when I say that I'm a CI babysitter people don't believe me. Here is the proof!

**Comment by ericmjl** [comment]
Thank you, @ocefpaf! I certainly believe you when you say that! :smile:

**Pull request merged**